install/install.py

```diff
@@ -142,6 +142,8 @@ def install(self, cmd_arguments):
         Colorizer.normal("Upgrading pip to the latest version ...")
         # Upgrade pip before install required libraries
         self.run_command("sudo pip2 install --upgrade pip")
+        # Upgrade setuptools
+        self.run_command("sudo pip2 install --upgrade setuptools")
```

Comment:
Add the `Colorizer.Normal("Upgrading setuptools to the latest version...")`  and  same for `cffi` below.